.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
       matrix:
         platform: [ubuntu-18.04]
-        os_name: [linux_openresty, linux_tengine, linux_apisix_master_luarocks, linux_apisix_current_luarocks, linux_openresty_mtls]
+        os_name: [linux_apisix_integrationtest]
```

Comment:
we can not use this way, you can add a new item `linux_apisix_integrationtest`

Reply:
got it and fixed